cmd/integration/integration.go

```diff
@@ -295,6 +302,9 @@ func podNotFound(c *client.Client, podNamespace string, podID string) wait.Condi
 func podRunning(c *client.Client, podNamespace string, podID string) wait.ConditionFunc {
 	return func() (bool, error) {
 		pod, err := c.Pods(podNamespace).Get(podID)
+		if apierrors.IsNotFound(err) {
```

Comment:
Unrelated errors should be logged since this is an integration test (it might be a bad configuration).  The odds of an individual call failing due to apiserver is so low that in a test case it's better to report it.

----- Original Message -----
> > @@ -295,6 +302,9 @@ func podNotFound(c *client.Client, podNamespace string,
> > podID string) wait.Condi
> >  func podRunning(c *client.Client, podNamespace string, podID string)
> >  wait.ConditionFunc {
> >  	return func() (bool, error) {
> >  		pod, err := c.Pods(podNamespace).Get(podID)
> > +		if apierrors.IsNotFound(err) {
> 
> If error terminates the poll, shouldn't we always return no err? What if the
> apiserver refuses the connection or some other unrelated errors happen?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/5775/files#r26977092